src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work");
```

Comment:
The message in `System.out.println` should be more informative and indicate the specific type of machine that is working. For example, it could be 'Excavator is working' to clearly specify the machine type.